fedora-toolbox

```diff
@@ -379,6 +428,30 @@ case $op in
         enter
         exit
         ;;
+    list )
+        list_images=1
+        list_containers=1
+        while [[ "$1" = -* ]]; do
+            case $1 in
+                -i | --images )
+                    list_containers=0
+                    ;;
+                -c | --containers )
+                    list_images=0
+                    ;;
+                * )
+                    exit_if_unrecognized_option $1
+            esac
+            shift
+        done
+        exit_if_extra_operand $1
+        if [ $list_images = 0 ] && [ $list_containers = 0 ]; then
+            list_images=1
+            list_containers=1
+        fi
+        list "$list_images" "$list_containers"
```

Comment:
I wonder if it will be more readable to have two separate `list_images` and `list_containers` functions.